site/viamguides/src/pomodoro-bot/pomodoro-bot.md

```diff
@@ -0,0 +1,439 @@
+<!-- ------------------------ -->
+## Integrate ENS160 Air Quality Sensor
+
+Duration: 10
+
+Let's integrate the ENS160 air quality sensor with your Pomodoro Bot to monitor indoor air quality effectively. The ENS160 sensor tracks VOCs (volatile organic compounds), eCO2 (equivalent carbon dioxide), and provides an Air Quality Index (AQI) – vital for maintaining an optimal workspace environment.
+
+![ENS160](assets/ens160.png)
+
+### Hardware Setup
+
+* **Power Connections:**
+    * Connect the `VCC` pin of the ENS160 sensor to the `3.3 Volts` pin on the Raspberry Pi.
+    * Connect the `Ground` pin of the ENS160 sensor to a `Ground` pin on the Raspberry Pi.
+
+* **I2C Communication:**
+    * Wire the `SCL` pin of the ENS160 sensor to the `SCL` pin on the Raspberry Pi.
+    * Wire the `SDA` pin of the ENS160 sensor to the `SDA` pin on the Raspberry Pi.
+
+Ensure all connections are secure and double-check your wiring to avoid any issues.
+
+### Software Integration
+
+* **Add the ENS160 Sensor to the Viam App:**
+    1. Open the Viam app.
+    2. Navigate to the **CONFIGURE** tab.
+    3. Add a new component for the ENS160 sensor and wait for the component to finish setup.
+    ![Add ENS160](assets/addENS160.png)
+
+* **Testing the Integration:**
+    1. Switch to the **Control** tab in the Viam app.
+    2. Observe the live data feed from the ENS160 sensor, including real-time air quality metrics such as VOC levels, eCO2 values, and AQI.
+      ![Test ENS160](assets/testENS160.png)
+
+<!-- ------------------------ -->
+## Integrate TEMT6000 Ambient Light Sensor
+
+Duration: 10
+
+Enhance your Pomodoro Bot by integrating the TEMT6000 ambient light sensor, which monitors light levels to help optimize workspace lighting for improved productivity. With this sensor, your bot can detect ambient light intensity and make adjustments as necessary to create a comfortable and efficient working environment.
+
+### Hardware Setup
+
+* **Components:**
+    * TEMT6000 Ambient Light Sensor
+      ![TEMT 6000](assets/temt6000.png)
+    * ADS1115 Analog-to-Digital Converter (ADC)
+      ![ADS1115 ADC](assets/ads1115.png)
+
+* **Connections:**
+    * **TEMT6000 to ADS1115:**
+        * Connect the analog output pin of the TEMT6000 to one of the analog input channels (e.g., A0) on the ADS1115.
+        * Connect the VCC pin of the TEMT6000 to the 3.3V or 5V power supply.
+        * Connect the Ground pin of the TEMT6000 to the Ground pin on the ADS1115.
+
+    * **ADS1115 to Raspberry Pi:**
+        * Connect the SCL pin of the ADS1115 to the SCL pin on the Raspberry Pi.
+        * Connect the SDA pin of the ADS1115 to the SDA pin on the Raspberry Pi.
+        * Connect the VCC pin of the ADS1115 to the 3.3V pin on the Raspberry Pi.
+        * Connect the Ground pin of the ADS1115 to a Ground pin on the Raspberry Pi.
```

Comment:
Please include physical pin numbers and a link to https://pinout.xyz/ as a reference for folks who are unfamiliar with the i2c pins on the Raspberry Pi.

Should they be connected to the I2C0 pins or the I2C1 (bus 0 or 1) on the Pi? Or is the connection chained together with the air quality sensor?